### DISCUSSION TITLE
Launching gk without arguments

### DISCUSSION THREAD

**Discussion by Zedb0T** [open-goal/jak-project#3589] [General]
Whe gk is launched without command line arguments, it should have some form of imgui input box where you can select/provide/change flags easily outside of terminal and then launch the game runtime with the selected configuration. This config should be saved to disk. This could be useful for quick debugging as well as being a less confusing experience for end users.

![image](https://github.com/user-attachments/assets/1a2f9ced-c024-42f2-8d6d-f7ff4869253c)

**Comment by xTVaser** [comment]
It'd be quite late in the startup process to do this with imgui, and if the concern is to ease things for end-users, that's probably not the way to go.  I've thought about this, and considered just using native dialogs (ie. provided by this library already in the repo https://github.com/native-toolkit/libtinyfiledialogs) 

However I abandoned the idea because in my opinion, at this point, launching via `gk` is just not something that will ever be better for the end-user.  Since they have to deal with extracting files properly, install the game (you could put a dialog here too), etc.  Not to mention niceties like updating and mods.  At some point you basically are just recreating the launcher experience within the game.
